packages/react-components/react-tree/src/components/Tree/useTree.ts

```diff
@@ -12,13 +14,94 @@ import type { TreeProps, TreeState } from './Tree.types';
  * @param ref - reference to root HTMLElement of Tree
  */
 export const useTree_unstable = (props: TreeProps, ref: React.Ref<HTMLElement>): TreeState => {
+  const {
+    isSubtree,
+    level: parentLevel,
+    openTrees: rootOpenTrees,
+    requestOpenChange: rootRequestOpenChange,
+    treeRef: parentTreeRef,
+  } = useTreeContext_unstable();
```

Comment:
shouldn't this be context selector? otherwise one subtree opening will open all other subtrees?